config.json

```
@@ -641,8 +641,8 @@
         "slug": "two-fer",
         "name": "Two Fer",
         "uuid": "57f02d0e-7b75-473b-892d-26a7d980c4ce",
-        "practices": [],
-        "prerequisites": [],
+        "practices": ["optional-parameters", "interpolation"],
+        "prerequisites": ["optional-parameters", "interpolation"],
```

According to the config.json file, the concept is `string-interpolation`:

```suggestion
        "practices": ["optional-parameters", "string-interpolation"],
        "prerequisites": ["optional-parameters", "string-interpolation"],
```

---

Like with my below comment, maybe guard against the `string-interpolation` not being unlocked by any Concept Exercise and add `strings` to the prerequisites?